src/components/ContentWidgets/TabView/TabView.css

```diff
@@ -0,0 +1,92 @@
+.tabs-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    list-style-type: none;
+    padding: 1rem;
+    background-color: #420DAB;
+    color: #ffffff;
+}
+
+.tab:hover {
+    color: #DAB420;
+    cursor: pointer;
+}
+
+.current-tab {
+    color: #DAB420;
+    border-bottom: 1px #DAB420 solid;
+}
+
+.current-tab-content, .current-tab-content-editor {
+    width: 100%;
+    border: 1px black solid;
+    margin-bottom: 5px;
+}
+
+.current-tab-content {
+    height: 15rem;
+    overflow: scroll;
+    padding: 3rem;
+    padding-top: 1rem;
+}
+
+.current-tab-content-editor {
+    display: flex;
+    justify-content: center;
+    padding: 1rem;
+}
+
+.current-tab-content-editor > textarea, .current-tab-content > ReactMarkdown {
+    width: 90%;
+    height: 5rem;
+}
+
+.add-tab-btn {
+    padding: 5px;
+    color: #ffffff;
```

Comment:
It might be worth adding this to a global color variable that you can access. I don't remember if it's in this specific project but I had a css file in my index file for a project where I set common variables I can access through any CSS file, so maybe a variable white or so. 

ie

```    
:root {
    --theme-color: #005696;
}
```

I realize I haven't done this, but it might be good to start doing that so we can remain consistent.

Reply:
Can you add to the `index.css` file or `App.css` file that selector and set a `--white` to be `#ffffff`? You will be able to access it by using `var(--white)`